enterprise/backend/src/metabase_enterprise/sso/integrations/sso_settings.clj

```diff
@@ -108,12 +107,10 @@ on your IdP, this usually looks something like http://www.example.com/141xkex604
   (deferred-tru "URL of JWT based login page"))
 
 (defsetting jwt-shared-secret
-  (deferred-tru "String used to seed the private key used to validate JWT messages")
-  :setter (fn [new-value]
-            (when (seq new-value)
-              (assert (u/hexadecimal-string? new-value)
-                       "Invalid JWT Shared Secret key must be a hexadecimal-encoded 256-bit key (i.e., a 64-character string)."))
-            (setting/set-value-of-type! :string :jwt-shared-secret new-value)))
+  (str (deferred-tru "String used to seed the private key used to validate JWT messages.")
+       " "
+       (deferred-tru "A hexadecimal-encoded 256-bit key (i.e., a 64-character string) is strongly recommended."))
```

Comment:
I think applying `str` forces the translation and that defeats the goal of `deffered-...`.

If the description is split into multiple lines just because otherwise it would be too long, then you can simply swap `deferred-tru` and `str`. In other words, you can have a form `(str ...)` with a bunch of string literals inside `deferred-tru`.

Reply:
That's a good point. This pattern I pulled from public settings, so it seems like the change should be made in a few places. For example:

https://github.com/metabase/metabase/blob/9dd96b3c68007ead23d2af055fe94e97234fef44/src/metabase/public_settings.clj#L120

Reply:
I think I'll create those changes in a separate PR, so as not to muddy this small PR with unrelated changes